### ISSUE TITLE
Question about results loss vs mAP

### ISSUE THREAD

**Issue by cyberphantom** [pierluigiferrari/ssd_keras#170]
Hi, I reproduce results of SSD model based on VGG16 weights and got training loss = 3.6559 and validation loss = 4.0241, then I ran the SSD validation ipynb and got mAP = 0.748 which is good. Then, I created a model and used the same settings without changes in the meta architecture (classification layers) of SSD and run the model on PascalVOC dataset and got training loss = 3.3216 and validation loss = 3.8345 which is much less than what I got in SSD model and it should be better, but when I ran SSD validation on my model I got mAP = 0.0, so is that due to a problem in training or is there any other setting that I'm not aware about?

**Comment by chuong98** [comment]
@cyberphantom We have a weird situation. When we reproduced the SSD-VGG16 with script ssd300_training.ipynb on PascalVOC, we got training_loss = 4.6 and val_loss = 4.3 after 120 epochs. The loss values look good, but mAP=0.43, which is much lower than the report.
May I know if you make any change to the code, or the data you used? Thanks in Advance.

**Comment by zhiqiangwan** [comment]
@chuong98 I have the same problem.
